packages/docs/src/components/preview/preview.tsx

```diff
@@ -62,7 +62,7 @@ const Preview: React.FunctionComponent<PreviewProps> = ({
 
 	return (
 		<Canvas className="csb-preview">
-			<div className="sui-wrap sui-theme--light">
+			<div id="sui-wrap" className="sui-wrap sui-theme--light">
```

Comment:
@creador-dev is it necessary to add the id here as we have added it to `.storybook/preview.tsx`?